plugins/signals/dev/index.tsx

```diff
@@ -18,18 +18,27 @@ import { createDevApp } from '@backstage/dev-utils';
 import { signalsPlugin } from '../src/plugin';
 import { Content, Header, Page } from '@backstage/core-components';
 import { Typography } from '@material-ui/core';
+import { useSignal } from '@backstage/plugin-signals-react';
+
+const SignalsDebugPage = () => {
+  const { lastSignal } = useSignal('debug');
+  return (
+    <Page themeId="home">
+      <Header title="Signals" />
+      <Content>
+        <Typography>Last signal:</Typography>
+        <Typography>
+          {lastSignal ? JSON.stringify(lastSignal) : 'Not received'}
```

Comment:
i know it's just a dev page, but making this monospaced, space preserving, and stringifying with indentation will probably make you happy :)

Feel free to use `<CodeSnippet>` from `core-components` as a helper for most of that